### PR TITLE
Add Edge metrics Vorto models.

### PR DESCRIPTION
**Pull request by dimitar-dimitrow**
Signed-off-by: Dimitar Dimitrov <dimitar.dimitrov3@bosch.io>

### CODE REVIEW

**Comment by dimitar-dimitrow** [comment]
@desislava-marinova, please wait for @e-grigorov to review the models before merging.

File: models/com.bosch.iot.suite.edge.metric-Filter-1.0.0.type/model.datatype
Comment (e-grigorov) [inline models/com.bosch.iot.suite.edge.metric-Filter-1.0.0.type/model.datatype]: It'll be helpful to say what will happen when it's skipped as it's described for the measure identifiers.

File: models/com.bosch.iot.suite.edge.metric-Measurement-1.0.0.type/model.datatype
Comment (e-grigorov) [inline models/com.bosch.iot.suite.edge.metric-Measurement-1.0.0.type/model.datatype]: `For group` should be: `For a group`

File: models/com.bosch.iot.suite.edge.metric-Metrics-1.0.0.fbmodel/model.functionblock
Comment (e-grigorov) [inline models/com.bosch.iot.suite.edge.metric-Metrics-1.0.0.fbmodel/model.functionblock]: `of metric data generation` should be: `of the metric data generation`

**Review by e-grigorov — CHANGES_REQUESTED** [review]

**Review by e-grigorov — APPROVED** [review]

**Pull request merged**
